### PR TITLE
New styles for inputs

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

- [x]  updates inputs and labels styles
- [x]  animations for label
- [x]  styleType prop for TextField and SelectField

### Screenshots

Screenshot 2019-04-29 at 15 35 34

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/components/SelectField/index.tsx [hunk 1]
```diff
@@ -20,9 +23,14 @@ type GenericSelectField<TValue> = React.StatelessComponent<
 
 const SelectField: GenericSelectField<SelectValue> = ({
   label = "",
+  styleType = "white",
   ...rest
 }) => (
-  <div className="react-select-wrapper">
+  <div
+    className={`react-select-wrapper ${
```
Comment (piotrgrundas) [inline src/components/SelectField/index.tsx]: use `classNames`

File: src/components/SelectField/scss/index.scss [hunk 2]
```diff
@@ -1,6 +1,19 @@
 @import "../../App/scss/variables.scss";
 
 .react-select-wrapper {
+  position: relative;
```
Comment (piotrgrundas) [inline src/components/SelectField/scss/index.scss]: empty line before each class, id etc

File: src/components/TextField/index.tsx [hunk 3]
```diff
@@ -4,33 +4,51 @@ import { FormError } from "../Form";
...
   extends React.InputHTMLAttributes<HTMLInputElement> {
   errors?: FormError[];
   helpText?: string;
   label?: string;
   iconLeft?: React.ReactNode;
   iconRight?: React.ReactNode;
+  styleType?: Style;
 }
 
+const generateClassName = ({ errors, iconLeft, styleType }: IClassNameArgs) => {
```
Comment (piotrgrundas) [inline src/components/TextField/index.tsx]: IMO no need param to be an object.

File: src/components/SelectField/scss/index.scss (hunk 2 above)
Comment (AlicjaSzu) [reply src/components/SelectField/scss/index.scss]: fixed

File: src/components/SelectField/index.tsx (hunk 1 above)
Comment (AlicjaSzu) [reply src/components/SelectField/index.tsx]: fixed

**Pull request merged**